src/lib/io/sol-iio.c

```diff
@@ -307,7 +316,53 @@ create_sysfs_trigger(struct sol_iio_device *device)
 }
 
 static bool
-check_trigger(struct sol_iio_device *device)
+create_hrtimer_trigger(struct sol_iio_device *device, const char *config_trigger_name)
+{
+    int success, len, r = false;
```

Comment:
Please, declare `r` as `bool`. It should avoid future misuse, if someone changes this code.